src/pages/index.tsx

```diff
@@ -1,9 +1,10 @@
 import Head from 'next/head'
+import Layout from '../components/layout'
 import styles from '../styles/Home.module.css'
 
-const Home = () => {
+const Home = (): JSX.Element => {
```

Comment:
Is TS not inferring return types?

Reply:
Too explicit & not necessary, you're right. Removed